pygeoapi/provider/mvt_postgres.py

```diff
@@ -0,0 +1,246 @@
+# =================================================================
+#
+# Authors:
```

Comment:
Please add yourself as author.  Example in https://github.com/geopython/pygeoapi/blob/master/setup.py#L3